### PR TITLE
GettingLogsAndWritingIntoFile

### LINKED ISSUE #27: Add Logs configuration

### PR DESCRIPTION
**Pull request by ebernevec**
Added logback.xml file for configuration of Logs

### CODE REVIEW

**Comment by MarcelMorozniuc** [comment]
Doesn't need there some adjustments in LogHook and log initialization ?

**Comment by MarcelMorozniuc** [comment]
Doesn't need there some adjustments in LogHook ?

**Comment by en-rmrosca** [comment]
Please test somewhere that the logs actually work

**Comment by ebernevec** [comment]
> Please test somewhere that the logs actually work

I've tested it, it works

**Comment by ebernevec** [comment]
> Doesn't need there some adjustments in LogHook ?

No, it doesn't . Only one row need to be added in each class where logs will be used, ex: 
static Logger log = LoggerFactory.getLogger(Class_Name.class);

**Comment by MarcelMorozniuc** [comment]
> > Doesn't need there some adjustments in LogHook ?
> 
> No, it doesn't . Only one row need to be added in each class where logs will be used, ex:
> static Logger log = LoggerFactory.getLogger(Class_Name.class);

https://github.com/vgangan-end/soa_5/pull/25/files there are some TODOs that need uncommented when logger is implemented

**Comment by margaritaq** [comment]
Please squash your commits

File: pom.xml [hunk 1]
```diff
@@ -28,39 +40,59 @@
             <version>5.7.0</version>
             <scope>test</scope>
         </dependency>
+
```
Comment (MarcelMorozniuc) [inline pom.xml]: no need empty lines between dependencies

Comment (ebernevec) [reply pom.xml]: Done

**Review by margaritaq — APPROVED** [review]

**Pull request merged**